lib/components/inputs/UploadField/UploadField.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import UploadField from './UploadField'
+
+const meta: Meta<typeof UploadField> = {
+  component: UploadField
+}
+export default meta
+
+type Story = StoryObj<typeof UploadField>
+
+export const Default: Story = {}
```

Comment:
Nice, maybe add some label/ info to your story so it will be shown by default?

Reply:
@udia76 good point, added those.